modules/channel-web/src/views/lite/main.tsx

```diff
@@ -259,7 +260,12 @@ class Web extends React.Component<MainProps> {
   }
 
   async playSound() {
-    if (this.state.played) {
+    // Preference for config object
+    const disableNotificationSound = isUndefined(this.config.disableNotificationSound)
+      ? this.props.config.disableNotificationSound
+      : this.config.disableNotificationSound
```

Comment:
isUndefined is not necessary. Simply use `this.config.disableNotificationSound === undefined`